mkdocs.yml

```diff
@@ -92,7 +92,12 @@ nav:
     - "Using Let's Encrypt": 'security/host-certs/lets-encrypt.md'
     - 'Using DigiCert': 'security/host-certs/digicert.md'
 - Releases:
-  - 'Release Notes': 'release/notes.md'
+  - OSG 3.5:
+      - 'Release Notes': 'release/notes.md'
+      - 'Updating to OSG 3.5': 'release/updating-to-osg-35.md'
```

Comment:
Did you forget to add this file?

Reply:
Oops, yes. I folded it in with the original commit and force pushed because I really wanted them together